### PR TITLE
reimagine12/data-formation-and-calculations

### PR DESCRIPTION
**Pull request by HannahSchafer**

### CODE REVIEW

File: src/components/Chart/Chart.js [hunk 1]
```diff
@@ -1,41 +1,152 @@
-import React from 'react';
-import './Chart.css';
```
Comment (jesseharold) [inline src/components/Chart/Chart.js:2]: I think we're gonna use this css file eventually

File: src/components/Chart/Chart.js [hunk 2]
```diff
@@ -1,41 +1,152 @@
...
-        backgroundColor: 'rgba(75,192,192,1)',
-        borderColor: 'rgba(0,0,0,1)',
-        borderWidth: 2,
-        data: [65, 59, 80, 81, 56]
-      }
-    ]
+
+export default class Chart extends Component {
+
+  state = {
```
Comment (jesseharold) [inline src/components/Chart/Chart.js:7]: You could do this using a map() of data, something like 

data.map(column => {state[column.id] = {amount: column.initial_amount, ..etc...}});

Comment (jesseharold) [reply src/components/Chart/Chart.js:7]: sorry, I should have said `setState`, not `state[x] = ...`

File: src/components/Chart/Chart.js (hunk 1 above)
Comment (HannahSchafer) [reply src/components/Chart/Chart.js:2]: ok cool, will add it back!

File: src/components/Chart/Chart.js (hunk 2 above)
Comment (jesseharold) [reply src/components/Chart/Chart.js:7]: @HannahSchafer I'm realizing this is more of a refactor than a change request,and I know you're busy with data stuff right now, so I'm going to approve, and perhaps add this refactor when I merge our code, if that's cool with you.

**Pull request merged**